azure-pipelines.yml

```diff
@@ -0,0 +1,164 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+variables:
+  # The $(FlutterToolPath) variable is generated by the FlutterInstall task 
+  # See code in the Flutter Build task https://github.com/aloisdeniel/vsts-flutter-tasks/blob/master/tasks/build/index.ts
+  flutterExecPath: $(FlutterToolPath)/flutter.bat
+
+trigger:
+  batch: 'true'
+  branches:
+    include:
+      - develop
+      - release
+      - master
```

Comment:
Do we want to trigger this CI / CD pipeline from `master` branch?

Reply:
All three - the first step is a CI so should run on all